Datastore/src/EntityMapper.php

```diff
@@ -189,6 +204,19 @@ public function convertValue($type, $value, $className = Entity::class)
                 break;
 
             case 'doubleValue':
+                if (is_string($value)) {
```

Comment:
Comment that it will enter this logic only when rest api values are given since gRPC values are always parsed correctly and converted to doubles. This would really help users understand  the default value of connectionType to `grpc`.

Reply:
Nice suggestion. This will go a long way.

I've added this.